3.5/aspnet/windowsservercore-1903/Dockerfile

```diff
@@ -10,7 +10,9 @@ RUN powershell -Command `
         Add-WindowsFeature Web-Asp-Net; `
         Remove-Item -Recurse C:\inetpub\wwwroot\*; `
         Invoke-WebRequest -Uri https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.6/ServiceMonitor.exe -OutFile C:\ServiceMonitor.exe `
-    && %windir%\System32\inetsrv\appcmd set apppool /apppool.name:DefaultAppPool /managedRuntimeVersion:v2.0
+    && %windir%\System32\inetsrv\appcmd set apppool /apppool.name:DefaultAppPool /managedRuntimeVersion:v2.0 `
+    &C:\Windows\Microsoft.NET\Framework64\v4.0.30319\ngen.exe update; `
```

Comment:
It looks like there is a missing `;` on the previous line.  That being said, I think for consistency with the 4.* Dockerfiles, ngen should be in a separate layer.

Reply:
PR updated, separate ngen in a new layer.